plugins/discovery/discovery.go

```diff
@@ -49,19 +50,21 @@ const (
 // started it will periodically download a configuration bundle and try to
 // reconfigure the OPA.
 type Discovery struct {
-	manager           *plugins.Manager
-	config            *Config
-	factories         map[string]plugins.Factory
-	downloader        bundle.Loader                       // discovery bundle downloader
-	status            *bundle.Status                      // discovery status
-	listenersMtx      sync.Mutex                          // lock for listener map
-	listeners         map[interface{}]func(bundle.Status) // listeners for discovery update events
-	etag              string                              // discovery bundle etag for caching purposes
-	metrics           metrics.Metrics
-	readyOnce         sync.Once
-	logger            logging.Logger
-	bundlePersistPath string
-	hooks             hooks.Hooks
+	manager              *plugins.Manager
+	config               *Config
+	factories            map[string]plugins.Factory
+	downloader           bundle.Loader                       // discovery bundle downloader
+	status               *bundle.Status                      // discovery status
+	listenersMtx         sync.Mutex                          // lock for listener map
+	listeners            map[interface{}]func(bundle.Status) // listeners for discovery update events
+	etag                 string                              // discovery bundle etag for caching purposes
+	metrics              metrics.Metrics
+	readyOnce            sync.Once
+	logger               logging.Logger
+	bundlePersistPath    string
+	hooks                hooks.Hooks
+	rawConfig            []byte
```

Comment:
Should we rename this to `rawBootConfig`, to not be misinterpreted as the raw _active_ config?

Reply:
NA. Updated to store the un-marshaled config now.